README.md

```diff
@@ -119,7 +116,37 @@ The following prerequisites are required:
 
 With your terminal go into the `coatrack/docker` directory where you will find the script `build-and-deploy-images-locally.sh` which builds CoatRack from source, creates container images on your local machine and runs CoatRack locally using these images. This is especially useful for developers who did changes to the source code and want to locally test the impact of these changes in a realistic setup.
 
-
+## Use FIWARE Keyrock for Identity Management instead of Github
+By default, CoatRack uses Github for identity management and login. As an alternative, you can set up CoatRack to use [Keyrock](https://keyrock-fiware.github.io/), which is the FIWARE enabler for Identity Management. In this case you have to set up Keyrock first and afterwards configure CoatRack to work with your Keyrock instance.
+The example set-up explained below is based on the following assumptions: 
+
+* Keyrock version 7.8.1 is deployed via Docker (image `fiware/idm:7.8.1`)
+* Keyrock is accessible via `localhost:3000`
+* CoatRack is accessible via `localhost:8080`
+
+Based on this, in order to make CoatRack work with the FIWARE Keyrock, you will need to:
+
+- At first you have to set-up/deploy your Keyrock instance. To do this, just follow the tutorial in https://fiware-idm.readthedocs.io/en/latest/getting_started/index.html
+- Please check that Keyrock is running correctly in your local machine by checking if you can access the Keyrock GUI,
+- If the Keyrock container is running and you can access the Keyrock GUI, then we need to start by registering CoatRack as application in Keyrock, to do this:
+  - Create a new user account or log in with the account `admin@test.com`, which is created as the administrator account by default,
+  ![KeyRock User Login](./spring-boot/admin/src/main/resources/static/images/keyrock/KeyRockLandingPage.png)
+  - After login, go to the application and click in the register button,
+  ![KeyRock register application](./spring-boot/admin/src/main/resources/static/images/keyrock/KeyRockRegisterApplication.png)
+  - Fill in all the fields in the registration form. The Callback URL must be `http:localhost:8080/login/aouth2/code/keyrock` as shown in the figure below. This is important because these values will be in the CoatRack configuration file later,
```

Comment:
```suggestion
  - Fill in all the fields of the registration form. The Callback URL must be `http:localhost:8080/login/aouth2/code/keyrock` as shown in the figure below. This is important because these values will be in the CoatRack configuration file later,
```

Reply:
Done